src/flashbar/__integ__/collapsible.test.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
_(Putting a random comment here so we can have a proper conversation thread)_

Here's my findings from testing the current implementation locally. I tested with VoiceOver together with Chrome and Safari.

1. The expanded flashbar items are not navigable as a list. Chrome+VO announces it as a list ("Notifications, 6 items") and Safari as a "group". But neither lets me use the VO commands to navigate the list item by item.
2. The tab order with the expand/collapse button is confusing. When collapsed, the button comes before the first (and only visible) flash item. When I expand, I would expect the the full list of items to appear after the button, but they are now "behind" the button.
3. I hear the Notifications counter quite often. I think it might be connected to too many things. Right now you will hear it via: 1) Live region updates, 2) `aria-describedby` on the expand/collapse button, 3) `aria-describedby` on the notifications group, 4) when navigating to the `span` that contains the counter text. Maybe one of the `aria-describedby` props is unnecessary?

Reply:
> 1. The expanded flashbar items are not navigable as a list. Chrome+VO announces it as a list ("Notifications, 6 items") and Safari as a "group". But neither lets me use the VO commands to navigate the list item by item.

I wonder if the same is true for the non-collapsible Flashbar, since AFAIK, except or the toggle button the markup for the list strictly speaking should be essentially the same. If so, I think anything affecting both versions can be handled separately for the sake of keeping the scope more focused. This PR is already very big and the API is not public yet.

Reply:
> 2. The tab order with the expand/collapse button is confusing. When collapsed, the button comes before the first (and only visible) flash item. When I expand, I would expect the the full list of items to appear after the button, but they are now "behind" the button.

This came from the a11y spec as well. I can share more details with you

Reply:
> I wonder if the same is true for the non-collapsible Flashbar

Yes, sorry, I should have clarified. This is happening for both types of flashbar. So this can be looked at outside of this PR.